e3db/src/main/java/com/tozny/e3db/Client.java

```diff
@@ -199,27 +217,29 @@
   private final byte[] privateSigningKey;
   private final byte[] publicSigningKey;
   private final StorageAPI storageClient;
-
   private final ShareAPI shareClient;
 
   private static final Charset UTF8 = Charset.forName("UTF-8");
 
   static {
+    anonymousClient = enableTLSv12(new OkHttpClient.Builder()).build();
```

Comment:
Did you rebase on top of changes we used for certificate pinning? The cert pinning work removed the anon client (as it was only used for registration). I remember you mentioning an anon client for uploading directly to S3; just wanted to be sure re-adding the client here was part of _that_ work and not a merge conflict from us working in parallel.

Reply:
I merged instead of rebasing (as this as already a PR) but otherwise yes - I added this back intentionally.